src/Stripe.net/Services/Events/EventUtility.cs

```diff
@@ -17,6 +17,22 @@ internal static readonly UTF8Encoding SafeUTF8
 
         public const int DefaultTimeTolerance = 300;
 
+        public static bool IsCompatibleApiVersion(string eventApiVersion)
+        {
+            // If the event api version is from before we started adding
+            // a release train, there's no way its compatible with this
+            // version
+            if (!eventApiVersion.Contains("."))
+            {
+                return false;
+            }
+
+            // versions are yyyy-MM-dd.train
+            var eventReleaseTrain = eventApiVersion.Split('.')[1];
+            var currentReleaseTrain = ApiVersion.Current.Split('.')[1];
```

Comment:
This needs to be the trimmed API version (i.e. the version without the beta version info)